pkg/asset/manifests/openstack/cluster.go

```diff
@@ -88,7 +88,7 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 			},
 		}
 	}
-	openStackCluster.SetGroupVersionKind(capo.GroupVersion.WithKind("OpenStackCluster"))
+	openStackCluster.SetGroupVersionKind(capo.SchemeGroupVersion.WithKind("OpenStackCluster"))
```

Comment:
it looks like this change caused the errors in https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_installer/9151/pull-ci-openshift-installer-master-e2e-openstack-ovn/1851695429587046400/artifacts/e2e-openstack-ovn/ipi-install-install/artifacts/.openshift_install-1730322510.log

```
time="2024-10-30T20:53:31Z" level=debug msg="E1030 20:53:31.731702    1938 controller.go:324] \"Reconciler error\" err=\"failed to update apiVersion in ref: failed to retrieve metadata from GVK resource: CustomResourceDefinition.apiextensions.k8s.io \\\"openstackclusters.\\\" not found\" controller=\"cluster\" controllerGroup=\"cluster.x-k8s.io\" controllerKind=\"Cluster\" Cluster=\"openshift-cluster-api-guests/k2m681my-d2a0e-mlvs7\" namespace=\"openshift-cluster-api-guests\" name=\"k2m681my-d2a0e-mlvs7\" reconcileID=\"5e77340c-7d83-418d-933e-0d59868c2865\""
```